Resources/Prototypes/_Ganimed/Recipes/Lathes/devices.yml

```
@@ -0,0 +1,26 @@
+- type: latheRecipe
+  id: BotanistBagOfHolding
+  result: BotanistBagOfHolding
+  completetime: 5
+  materials:
+    Steel: 2000
+    Silver: 750
+    Plasma: 1500
+    Uranium: 150
```

По идее блюспейс штуки должны иметь блюспейс кристаллы в рецепте.
```suggestion
    Uranium: 150
    ADTBScrystal: 500
```